backend/src/handlers/helpful_links_handler.go

```diff
@@ -7,6 +7,8 @@ import (
 	"net/http"
 	"net/url"
 	"strconv"
+
+	"github.com/sirupsen/logrus"
```

Comment:
we use `log` for the variable name of the `sLog` wrapper in our custom handlers, so it would conflict. I'm beginning to wish I never alias'ed logrus in the first place :/ we should slowly change it back